benchmark/syclblas/blas1/nrm2.cpp

```diff
@@ -99,12 +100,12 @@ void run(benchmark::State& state, ExecutorType* executorPtr, index_t size,
 }
 
 template <typename scalar_t>
-void register_benchmark(blas_benchmark::Args& args, ExecutorType* exPtr,
+void register_benchmark(blas_benchmark::Args& args, blas::SB_Handle* exPtr,
```

Comment:
Can we also rename the variables `exPtr` to something else like `sbHandlePtr`?

Reply:
done